java/KinesisConnectors/src/main/resources/flink-application-properties-dev.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "PropertyGroupId": "InputStream0",
+    "PropertyMap": {
+      "stream.arn": "arn:aws:kinesis:us-east-1:060795913058:stream/ExampleInputStream",
```

Comment:
Please do not push real resource ARN. 
Use placeholders instead (not fictional ARN). This gives a more obvious failure when the user tries to run it locally without modifying the config